src/ext/posix_fd_support.rs

```diff
@@ -4,24 +4,25 @@ use std::os::raw::c_char;
 
 pub const CLAP_EXT_POSIX_FD_SUPPORT: *const c_char = b"clap.fd-support\0".as_ptr() as *const c_char;
 
-pub const CLAP_FD_READ: clap_fd_flags = 1 << 0;
-pub const CLAP_FD_WRITE: clap_fd_flags = 1 << 1;
-pub const CLAP_FD_ERROR: clap_fd_flags = 1 << 2;
+pub const CLAP_POSIX_FD_READ: clap_posix_fd_flags = 1 << 0;
+pub const CLAP_POSIX_FD_WRITE: clap_posix_fd_flags = 1 << 1;
+pub const CLAP_POSIX_FD_ERROR: clap_posix_fd_flags = 1 << 2;
 
-pub type clap_fd_flags = i32;
+pub type clap_posix_fd_flags = i32;
```

Comment:
Should be `u32`: https://github.com/free-audio/clap/blob/469541c05a0fd59f123422dda3d792ee0f73bc5f/include/clap/ext/posix-fd-support.h#L21

Reply:
Thanks,  fixed! (In the other PR)